addon/components/semantic-forms/instance-table.hbs

```diff
@@ -8,29 +8,48 @@
       <AuHeading @skin="2">{{this.tableTitle}}</AuHeading>
     </Group>
     <Group>
-      <AuButtonGroup>
+      {{! template-lint-disable require-context-role }}
+      <AuDropdown
+        @title="Meer acties"
+        @skin="secondary"
+        @icon="chevron-down"
+        @iconAlignment="right"
+        @alignment="center"
+      >
+        {{yield to="extraActions"}}
         {{#if @labels}}
           <AuButton
-            @skin="secondary"
+            role="menuitem"
+            @skin="link"
             {{on "click" (fn (mut this.isColumnConfigurationOpen) true)}}
           >
             Configureer kolommen
           </AuButton>
         {{/if}}
         {{#if @showDownloadButton}}
           <AuButton
-            @skin="secondary"
             {{on "click" (fn (mut this.isModalOpen) true)}}
           >Download</AuButton>
         {{/if}}
+      </AuDropdown>
+      <AuButtonGroup>
         <AuLink
-          @route="forms.form.new"
+          @route={{this.newFormRoute}}
           @model={{this.formInfo.formDefinition.id}}
           @skin="button"
         >Voeg item toe</AuLink>
       </AuButtonGroup>
     </Group>
   </AuToolbar>
+  <AuToolbar
+    @size="large"
+    class="au-u-padding-bottom-none au-u-margin-bottom"
+    as |Group|
+  >
+    <Group>
+      <p>{{this.tableDescription}}</p>
+    </Group>
+  </AuToolbar>
   <AuToolbar
```

Comment:
while you're at it, can you change the placeholder in the search below? it contains the id and it looks very weird. better to just have 'Zoek...' as a placeholder